mlflow/xgboost.py

```diff
@@ -398,12 +398,12 @@ def log_feature_importance_plot(features, importance, importance_type):
         # logging feature importance as artifacts.
         for imp_type in importance_types:
             imp = model.get_score(importance_type=imp_type)
```

Comment:
Would it hurt to put this line in the try/catch block as well? Seems like this could potentially fail.

Reply:
agreed, I don't think it would hurt. The docs for xgboost do not indicate a contract for the return value of this function